### PR TITLE
Added SRE meeting types to db and graph

### PR DESCRIPTION
**Pull request by Muncherkin**
Added database and graph support for SRE meetings of type "grundnivå", "avancerad nivå" and "intern". 

Other than the main feature: 
- Added myself to the contributor list 
- Removed two broken tests from `user.api.ts` which I have spoken to Froborg about
- Fixed an error message that was missing the word "har"
- Redid the name assignment logic in  the meeting reducer
- Fixed the prisma:reset commad for Windows users (without destroying it for linux chads)

Note this change is dependant on PR #136 in efterphest.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "Emil Eriksson <eje1999@gmail.com>",
     "Axel Froborg <axel.froborg@gmail.com>",
     "Marcus Lindell <lindellmarcus99@gmail.com>",
-    "Eric Weidow <eric@weidow.se>"
+    "Eric Weidow <eric@weidow.se>",
+    "Axel Anderssin <axel.so.andersson@gmail.com>"
```
Comment (afroborg) [inline package.json:34]: Nice

File: src/reducers/meeting.reducer.ts [hunk 2]
```diff
@@ -19,11 +19,11 @@ const addIfRefNotNull = (
 
 export function meetingReduce(meeting: PrismaMeeting): MeetingResponse {
   let name: string;
-
-  if (meeting.type === MeetingType.Sm || meeting.type === MeetingType.Extra) {
-    name = `${meeting.type}${zeroPad(meeting.number, 2)}`;
-  } else {
+  const guildMeetings = [MeetingType.Htm, MeetingType.Vtm, MeetingType.Vm, MeetingType.Extra];
```
Comment (afroborg) [inline src/reducers/meeting.reducer.ts:22]: move this to the global scope instead so it's not redefined on every function call

**Review by afroborg — CHANGES_REQUESTED** [review]

File: src/reducers/meeting.reducer.ts (hunk 2 above)
Comment (Muncherkin) [reply src/reducers/meeting.reducer.ts]: True and based. Did it!

File: package.json (hunk 1 above)
Comment (Muncherkin) [reply package.json]: I should know how to spell my last name I suppose. Oh well...

**Review by afroborg — APPROVED** [review]

**Pull request merged**